v2/support/secure_settings_sample_app/README.md

```diff
@@ -0,0 +1,19 @@
+# App name
+
+Bag End Catering Helper
```

Comment:
I think this README file needs a bit more polishing to be useful 😄You can see other READMEs of current sample apps for inspiration.